requirements.yml

```diff
@@ -10,6 +10,7 @@
   version: v1.2.2
 - src: Stouts.grafana
   version: 2.0.1
+- src: retr0h.etcd
```

Comment:
Could you pin the (latest) version of this so that we don't unwillingly upgrade?